src/elements/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { TouchableOpacity, TouchableWithoutFeedback } from 'react-native';
+import { CheckBox } from 'react-native-elements';
```

Comment:
Unforunately, I think using this library and this imported component isn't a great choice for our checkbox. 14 / 20 props are irrelevant and/or redundant to what our checkbox needs to be and to how our checkbox can be implemented. If this was just a placeholder, `react-native` already comes with a CheckBox that could be used.

Though, it seems like a useful library for projects that are using other font and UI systems.

These two produce the same results:
```tsx
<TouchableHighlight
   onPress={() => { !disabled && handlePress(); }}
>
   <Icon
      name={isChecked ? 'arrowDown' : 'arrowUp'}
      color={color}
      size={size}
   />
</TouchableHighlight>

<CheckBox
   checkedIcon={<Icon name="arrowDown" color={color} size={size} />}
   uncheckedIcon={<Icon name="arrowUp" color={color} size={size} />}
   Component={disabled ? TouchableWithoutFeedback : TouchableHighlight}
   checked={isChecked}
   onPress={handlePress}
   size={size}
/>
```

Reply:
I did see that react-native had a CheckBox component but the documentation online said that it was no longer supported, so I didn't think that made it a viable option. Should I change it to the react-native component anyways?

Reply:
Oh nice find on the deprecation, I didn't see that in the in-file documentation. That would have just been a convenient placeholder, all without needing to add an extra dependency.

I think that a component structure that just uses the native `TouchableHighlight` and our custom Icon component is a better choice. I don't see the need to depend upon an entire library if we can accomplish it on our own and with only a handful of lines of code.